index.html

```diff
@@ -350,10 +350,14 @@ <h2 style="text-align:right;">Contributed by</h2>
     </p>
   </footer>
 
+  <script src="./js/libs/jsonld/jsonld.min.js"></script>
   <script>
     initFieldsData();
-    initCallbacks();
-    loadStateFromStorage();
+    initContext().then(contexts => {
+      initJsonldLoader(contexts);
+      initCallbacks();
+      loadStateFromStorage();
+    });
```

Comment:
I hadn't thought of the order here. That's perfect

Reply:
> I hadn't thought of the order here. That's perfect

Weirdly, it was failing randomly because of this order, I have changed it back!